src/App.js

```diff
@@ -1,129 +1,53 @@
-        };
-
-
-        try {
-            const response = await fetch(url, options);
-            if (!response.ok) {
-                throw new Error(`Error: ${response.status}`);
-            }
-
-            const data = await response.json();
-            
-            // Mapping data
-            const todos = data.records.map(record => ({
-                title: record.fields.title,
-                id: record.id,
-                completed: record.fields.completed || false,
-            }));
-
-            setTodoList(todos); // Set todoList
-
-        } catch (error) {
-            console.error('Fetch error:', error.message);
-        } finally {
-            setIsLoading(false);
-        }
-    }, [url]);
-
-    useEffect(() => {
-        fetchData();
-    }, [fetchData]);
-
-    const addTodo = async (title) => {
-        const newTodo = {
-            fields: {
-                title: title, // Airtable field name
-            }
-        };
-
-        const options = {
-            method: 'POST',
-            headers: {
-                'Authorization': `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
-                'Content-Type': 'application/json'
-            },
-            body: JSON.stringify(newTodo)
-        };
-        try {
-            const response = await fetch(url, options);
-            if (!response.ok) {
-                throw new Error(`HTTP error! status: ${response.status}`);
-            }
 
-            const data = await response.json();
-            setTodoList([...todoList, {
-                title: data.fields.Title,
-                id: data.id,
-                completed: data.fields.completed || false,
-            }]);
-
-        } catch (error) {
-            console.error('Error adding todo:', error);
-        }
-    };
-
-    const removeTodo = (id) => {
-        const updatedList = todoList.filter(todo => todo.id !== id);
-        setTodoList(updatedList);
-    };
-
-    const toggleTodoCompletion = (id) => {
-        const updatedTodoList = todoList.map((todo) =>
-            todo.id === id ? {...todo, completed: !todo.completed} : todo
-        );
-
-        const sortedTodoList = updatedTodoList.sort((a, b) =>
-            a.completed === b.completed ? 0 : a.completed ? 1 : -1
-        );
-
-        setTodoList(sortedTodoList);
-    };
-
-    const reorderTodo = (newTodoList) => {
-        setTodoList(newTodoList);
-    };
+const TableChooser = () => {
+    const [tableName, setTableName] = useState(process.env.REACT_APP_TABLE_NAME);
+    return (
+        <div>
+            <select
+                id="selectField"
+                value={tableName}
+                onChange={(event) => setTableName(event.target.value)}
+            >
+                <option value={process.env.REACT_APP_TABLE_NAME}>Todo List</option>
+                <option value="List2">Todo List 2</option>
+            </select>
+            <TodoContainer tableName={tableName}/>
+        </div>
+    );
+};
```

Comment:
I think this would be better off in its own file honestly. Also if you wanted to make this more dynamic, on Airtable you can click on the down arrow on the Default table name and select duplicate table. Since it creates another table with the same fields you could then add another secret and access it like `<option value={process.env.REACT_APP_TABLE_NAME_2}>` or something like that.